packages/graalvm/src/test/kotlin/elide/runtime/gvm/internals/js/node/NodeFsTest.kt

```diff
@@ -32,10 +32,11 @@ import elide.runtime.intrinsics.js.node.path.Path as NodePath
 
 /** Tests for Elide's implementation of the Node `fs` built-in module. */
 @TestCase internal class NodeFsTest : NodeModuleConformanceTest<NodeFilesystemModule>() {
-  override val moduleName: String get() = "fs"
-  override fun provide(): NodeFilesystemModule = NodeFilesystemModule()
   @Inject lateinit var filesystem: NodeFilesystemModule
 
+  override val moduleName: String get() = "fs"
+  override fun provide(): NodeFilesystemModule = filesystem
+
```

Comment:
Same as previous comment, need to use the injected value instead of a manually constructed one.

Reply:
This was only a temporary measure until injection was needed, yes.